Examples/Along-slope-velocities.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>Can delete "example" in  title. I think we've been trying to make the titles the same as the notebook filenames.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/420/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>